django_airavata/apps/groups/templates/django_airavata_groups/groups_manage.html

```diff
@@ -0,0 +1,107 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<h1 align="center">Manage Groups</h1>
+<br>
+<div class="container">
+    {% if user.is_authenticated %}
+    <div class="row">
+        <a href="{% url 'django_airavata_groups:create' %}">
+            <button type="button" class="btn btn-default toggle-add-tenant"><span
+                        class="glyphicon glyphicon-plus"></span>Create a New Group
+            </button>
+        </a>
+    </div>
+    <br>
+    <hr>
+    {% if messages %}
+    <div class="alert alert-info">
+        <ul class="messages">
+            {% for message in messages %}
+            <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
+            {% endfor %}
+        </ul>
+    </div>
+    {% endif %}
+    <br>
+    <div class="row">
+        <div class="col-md-12 table-responsive">
+            <h3>Groups you own</h3>
+            <br>
+            <br>
+            <table class="table table-striped table-bordered">
+                <thead>
+                    <tr>
+                        <th style="text-align:center">GROUP_ID</th>
+                        <th style="text-align:center">NAME</th>
+                        <th style="text-align:center">DOMAIN_ID</th>
+                        <th style="text-align:center">OPTIONS</th>
+                    </tr>
+                </thead>
+                <tbody align="center">
+                    {% for group in owner %}
+                    <tr>
+                        <td>{{ group.groupId }}</td>
+                        <td>{{ group.name }}</td>
+                        <td>{{ group.domainId }}</td>
+                        <td>
+                            <a href="{% url 'django_airavata_groups:view' %}?group_id={{ group.groupId }}">
```

Comment:
Need to URL encode the groupId, also on lines 52 and 55. You should be able to do `{{ group.groupId | urlencode }}`.